.gitignore

```diff
@@ -1,31 +1,4 @@
-# Logs
-logs
 *.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-dist
-
-# Dependency directory
-# Commenting this out is preferred by some people, see
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+.DS_Store
 node_modules
-
-# Users Environment Variables
-.lock-wscript
-/.idea/
-.vscode
\ No newline at end of file
+dist
```

Comment:
Why was `.npmignore` removed? Should we not ignore ts files and config files for npm and only publish lib and package files?

Reply:
It is my understanding that this is how the cool kids do it nowadays ?
Publish `esm`, `CJS`, `minified` and everything ?
And let the consumer decide what to use ? At least this is how it was set up from `tsdx`

Reply:
Yes. esm, CJS, minified and all will be in the lib dir, right? Is there a point for the src files and such to be in npm? Haven't seen a lot of cool kids publish their src.